lib/jsdom/living/nodes/ElementCSSInlineStyle-impl.js

```diff
@@ -1,19 +1,32 @@
 "use strict";
-const cssstyle = require("cssstyle");
+
+const CSSStyleProperties = require("../generated/CSSStyleProperties");
+const idlUtils = require("../generated/utils");
 
 class ElementCSSInlineStyle {
   _initElementCSSInlineStyle() {
     this._settingCssText = false;
-    this._style = new cssstyle.CSSStyleDeclaration(newCssText => {
-      if (!this._settingCssText) {
-        this._settingCssText = true;
-        this.setAttributeNS(null, "style", newCssText);
-        this._settingCssText = false;
+    this._list = [];
```

Comment:
I still don't understand what `this._list` is and why we need to copy a bunch of stuff from `this._style` into `this._list`.

Reply:
Done https://github.com/jsdom/jsdom/pull/3957/commits/c07acad6ecbccdeaaae26b01972383996b57662d